components/docusign/common.mjs

```diff
@@ -118,6 +118,51 @@ export default {
         config,
       });
     },
+    async getTemplate(baseUri, templateId) {
+      const config = {
+        method: "GET",
+        url: `${baseUri}templates/${templateId}`,
+      };
+      return this._makeRequest({
+        config,
+      });
+    },
+    async listTemplateCustomFields(baseUri, templateId) {
+      const config = {
+        method: "GET",
+        url: `${baseUri}templates/${templateId}/custom_fields`,
+      };
+      return this._makeRequest({
+        config,
+      });
+    },
```

Comment:
I don't see where the method `listTemplateCustomFields` is being used.

Reply:
It isn't, I've left it there in case it's needed in the future